packages/snapshot-tests/jest.config.js

```diff
@@ -1,17 +1,21 @@
-// Test env in dev mode produces a big amount of stdin/out listeners, so limit increased
-require('events').EventEmitter.defaultMaxListeners  = 50;
+import {EventEmitter} from 'events';
+import {rimraf} from 'rimraf';
 
+// Test env in dev mode produces a big amount of stdin/out listeners, so limit increased
+EventEmitter.defaultMaxListeners = 50;
 // Cleanup diff output directory
-const rimraf = require('rimraf');
 rimraf.sync('./.diff');
+// Enable ESM modules in Jest
+process.env.NODE_OPTIONS = '--experimental-vm-modules';
 
 // Actual Jest configuration
-module.exports = {
+export default {
   preset: 'jest-puppeteer',
   transform: {
-    '^.+\\.tsx?$': 'ts-jest',
+    '^.+\\.tsx?$': '@swc/jest',
```

Comment:
Probably, after all the other modifications we have, the global `ts-jest` will work here:

```suggestion
    ['ts-jest', { useESM: true }],
```

I didn’t properly clarify my point earlier (on weakly meeting), sorry about that. What I meant was: if swc solves our problems or at least simplifies something with ESM, then I’m open to using it here. Otherwise, there’s no real benefit to using it.

The only general advantage of swc is its speed. However, in this case, we’re not focusing on performance, and it won’t provide any noticeable improvements. 
PS: We can discuss it on a call detailed just in case.

Reply:
I don't insist on using swc. The only reason it was utilized is because tests were failing in any other configuration. But now it does, even though I was removing cache every single time.